src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/opt.cpp

```diff
@@ -1115,7 +1116,7 @@ HostGatherDQ::HostGatherDQ(Context::Ref ctx) {
         const auto& matched_out_qweight = node_to_output.at(qweight);
         auto qweight_type = matched_out_qweight.get_element_type();
 
-        if (out_len >= 2048 && qweight_type == ov::element::i4) {
+        if (out_len >= 2048 && qweight_type == ov::element::i4 || qweight_type == ov::element::i8) {
```

Comment:
`qweight_type == ov::element::i4 || qweight_type == ov::element::i8` should be in the brackets right?

Reply:
good catch, yeah